src/OmniSharp.Host/Program.cs

```diff
@@ -30,6 +30,7 @@ public static void Main(string[] args)
             var otherArgs = new List<string>();
             var plugins = new List<string>();
             var serverInterface = "localhost";
+            var encoding = System.Text.Encoding.Default;
```

Comment:
Since Encoding.Default is not available on .NET Core, it's probably best to just not set the encoding if the `--encoding` is not specified. So, set `encoding` to null here and then check for null later. E.g.

```C#
Encoding encoding = null;
```